README.md

```diff
@@ -0,0 +1,120 @@
+wish to post to Facebook as Instant Articles, the simplest way is to create a
+feed from your Drupal site and then have your Facebook page admin configure
+the Instant Articles options to point to your website to ingest the feed.
+
+You'll need to configure which content types you want to be allowable as Instant 
+Articles, then configure a view mode to specify which fields you want to map to
+which region of the Instant Article format, then you're all ready to create the 
+feed.
+
+To create the actual feed, there are two main options: a simple RSS feed, or a 
+more advanced feed using the [Views module](https://www.drupal.org/project/views). 
+
+#### RSS feed (simple)
+
+If you enable the fb_instant_articles_rss.module you will have a feed created 
+for you automatically. You will then have the option to either add all published 
+content of one or more types to the feed, or else select individual pieces of 
+content to be included in the feed. 
+
+#### Views feed (advanced)
+
+Alternately, if you enable the fb_instant_articles_views.module you will be able
+to choose more advanced filtering rules for which content is included in the 
+feed. Currently only a rendered view mode (not a fields view) is supported.
+
+### 2. API method
+  
+If you're a publisher who wants more control over exact timing of publishing - 
+editing, modifying public access, or removing content from your Facebook Instant 
+Articles library - the API approach allows for greater control.
+
+#### API Rules integration (optional)
+
+If you're using the API you will need to determine under what conditions the API
+creates, edits or removes content from Facebook's Instant Articles library. This
+submodule adds actions (both Drupal core actions and [Rules](https://drupal.org/project/rules) 
+module actions) which can be triggered based on your chosen configurations. 
+Alternately, If you do not choose to use this module to trigger API actions, you 
+will want to trigger them some other way (via a custom module implementing 
+entity CRUD hooks checking your own custom conditions in code, etc).
+
+#### API Report (optional)
+
+Optionally enable this module to give a report of API interactions, so that for 
+example any content that fails Facebook Instant Article format validation will
+be easier to find and fix.
+
+## Dependencies
+
+The Drupal Facebook Instant Articles modules have different dependencies, see 
+"About Drupal integration" above to help choose which modules are right for you.
+
+- Base (fb_instant_articles.module)
+    - Depends on:
+        - [Facebook Instant Articles PHP SDK](https://github.com/facebook/facebook-instant-articles-sdk-php) 
+provides necessary support for ensuring Drupal content is added to facebook in 
+the required format for Instant Articles. 
+    - Optionally (recommended) integrates with:
+        - [Composer Manager](https://www.drupal.org/project/composer_manager) for 
```

Comment:
Make this more explicit, so users know they should enable composer manager unless they really REALLY know what they're doing (currently /Logger doesn't load and blows up).